cartography/intel/gcp/dataflow.py

```diff
@@ -0,0 +1,186 @@
+import json
+import logging
+import time
+from typing import Dict
+from typing import List
+
+import neo4j
+from googleapiclient.discovery import HttpError
+from googleapiclient.discovery import Resource
+from cloudconsolelink.clouds.gcp import GCPLinker
+
+from . import label
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+gcp_console_link = GCPLinker()
+
+
+@timeit
+def get_dataflow_jobs(dataflow: Resource, project_id: str, regions: list, common_job_parameters) -> List[Dict]:
+    jobs = []
+    try:
+        req = dataflow.projects().jobs().list(projectId=project_id)
```

Comment:
To list the all jobs across all regions, use projects.jobs.aggregated. Using projects.jobs.list is not recommended, as you can only get the list of jobs that are running in us-central1
https://cloud.google.com/dataflow/docs/reference/rest/v1b3/projects.jobs/list